packages/react/src/components/Backdrop/Backdrop.stories.mdx

```diff
@@ -0,0 +1,92 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import {useArgs} from '@storybook/client-api';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import Backdrop from './Backdrop.tsx';
+import Button from '../Button/Button.tsx';
+
+export const meta = {
+  component: Backdrop,
+  title: StoryConfig.Backdrop.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => {
+  const [{open}, updateArgs] = useArgs();
+  return (
```

Comment:
Let's keep a space after the `return` statement

Reply:
Please refer https://github.com/wso2/oxygen-ui/pull/142#discussion_r1236250736